src/main/webapp/script.js

```diff
@@ -100,7 +141,6 @@ function fetchAuthentication() {
         // If logged in user is not registered, redirect to signup page.
         window.location.replace(user.authenticationUrl);
       }
-
```

Comment:
Why did you delete this line? (Not asking you to add it back, all I want to know is if there is a guideline I must follow for blank lines.)

Reply:
Mmmm I don't remember erasing it on purpose. I usually don't leave blanks for file size motives but I usually dont mess with other people's code, sorry!